test/singleton/cart_singleton_one_node.yaml

```diff
@@ -24,14 +24,14 @@ hosts: !mux
     cli:
       - boro-A
 tests: !mux
-  singleton:
-    name: singleton_basic
-    srv_bin: tests/crt_echo_srv
-    srv_arg: ""
-    srv_env: ""
-    srv_ppn: "1"
+  #singleton:
```

Comment:
Lots of things commented out...any reason not to delete?   Assuming we no longer support singleton attach ?